plugins/updater/guest-js/index.ts

```diff
@@ -68,6 +90,15 @@ class Update extends Resource {
       rid: this.rid,
     });
   }
+
+  async close(): Promise<void> {
+    if (this.downloadedBytesRid !== undefined) {
+      await invoke("plugin:resources|close", {
+        rid: this.rid,
```

Comment:
```suggestion
        rid: this.downloadedBytesRid,
```

I guess it is better to add back your `DownloadedBytes` class and use its close method instead